pkg/workflows/wasm/host/module.go

```diff
@@ -580,7 +587,49 @@ func runWasm[I idMessage, O proto.Message](
 }
 
 func (m *module) SetCapabilityExecutor(handler CapabilityExecutor) error {
```

Comment:
Something seems off to me. Why is it in the mod config and also something you can set later?